### DISCUSSION TITLE
Welcome to ngx-daterangepicker-material Discussions!

### DISCUSSION THREAD

**Discussion by fetrarij** [fetrarij/ngx-daterangepicker-material#463] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by vamdandu** [comment]
Ngx-Date-range-picker-material is not displaying properly and picker selection also not working with angular 17 version.I tried overriding z-index also.The date range picker is getting collapsed with other controls in the page when it displays.
